apps/blog/content/posts/what-is-memebattle.en.mdx

```diff
@@ -7,8 +7,106 @@ tags: ['MemeBattle']
 author: 'themezv'
 ---
 
-## Started
+## How it all began
 
+In 2017, a group of students from Novosibirsk State Technical University teamed up to participate in an online hackathon.
+The hackathon was about entertainment and fun. We created an application (web and Android) with an infinity battle of memes. Every few seconds users saw a pair of memes and voted for one of them.
+Players who rated the meme with a majority vote received the internal currency – MemeCoin.
+We won that hackathon and received investments for development.
+
+<details className="my-6">
+  <summary>First application version screenshots</summary>
+  <img src="/content-images/what-is-memebattle/memebattle_v1.png" alt="MemeBattle v1. Screenshow of web application"/>
+  <img src="/content-images/what-is-memebattle/memebattle-playmarket.png" alt="MemeBattle v1. Screenshot of application page on google play"/>
+</details>
+
+After that we have participated in hackathons several years. Local hackathons in Siberia (Tomsk, Kemerovo, Novosibirsk) and grand ones in Moscow – it was our hobby.
+Meanwhile, we periodically returned to the original idea, but didn't create real product 😢
+
+<InstagramPost publicationId="BaRdqExl35J" description="A post shared by Themezv (@artem_zverev.9)" />
+
+In parallel with hackathons we have done some education projects. For example, in union with other students, organized IT community AVTF – few meetups by different streams (web development, mobile development, databases, information security, data science). Just so you know, on the first meetup were more than 150 people!
+
+<Image src='/content-images/what-is-memebattle/ITC2.jpg' alt="Photo from first meetup of IT community AVTF" width={1920} height={925} />
+
+We often discussed about MemeBattle and told to friend. But we could never answer the question "What is MemeBattle". Usually we respond like:
```

Comment:
We often discussed MemeBattle and told our friends about it. But we have not been able to answer the question "What is MemeBattle". We usually answer like this: